src/main/java/com/google/sps/Utility.java

```diff
@@ -203,6 +205,80 @@ public static MultiMutation getMultiMutationAtIndex(List<MultiMutation> multiMut
     return lst;
   }
 
+  /**
+   * Returns a list of the indices of the mutations in origList that mutate
+   *
+   * @param nodeName the name of the node to filter
+   * @param origList the original list of mutations
+   * @param tokenName paramter for token name. list ends when that token is deleted
+   * @return a list of indices that are relevant to the node, truncated at the point a given token
+   *     is deleted
+   */
+  public static ArrayList<Integer> getMutationIndicesOfNode(
```

Comment:
Do we need this function anymore?

Reply:
it's used in `findRelevantMutations` which we do use

Reply:
This is the one with the additional token parameter which I don't think is ever called?